USECASES.md

```diff
@@ -1,5 +1,15 @@
 # Use Cases
 
+## Provisioning Models
+
+- **Network Peripheral**: Peripheral to the CPU exposing a network interface to the CPU OS, provisioned directly by the CPU OS.
+
+- **Data Processor Peripheral**: Peripheral to the CPU exposing a data processor (DPU) interface to the CPU OS, provisioned by the platform BMC.
```

Comment:
Peripheral to host CPU exposing a data processor (DPU) interface to the host CPU OS, provisioned by the host CPU or platform/host BMC.

Is your assumption here that the platform/host BMC can not be in a seperate trust domain from host CPU?

Reply:
First- are you asking can we add ‘by the host CPU or’ by the platform BMC?  I’m fine with adding this or, I had left it out because it felt like potentially a fifth model, or maybe the host CPU case was better meant for the NetworkPeripheral case.

Second Question- I don’t really understand, I don’t think there is any assumption on trust boundary between host and platform BMC.

Reply:
I was trying to clarify the definintion.

**Data Processor Peripheral**: Peripheral to **host** CPU exposing a data processor (DPU) interface to the **host** CPU OS, provisioned by the **host CPU** or platform/host BMC.

I think what your saying is that in this model the host has full control of the DPU.  I think of this as trusted host mode.  The DPU is provisioned through the host.

I was asking the question why did you put "provisioned by the platform BMC" in this model and not in "Infrastructure Device".  Why does the host <-> DPU relationship change the platform BMC <-> DPU relationship?